example/ios/RealmTsTemplate.xcodeproj/project.pbxproj

```diff
@@ -376,25 +375,6 @@
 			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-RealmTsTemplate-RealmTsTemplateTests/Pods-RealmTsTemplate-RealmTsTemplateTests-resources.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
-		FD10A7F022414F080027D42C /* Start Packager */ = {
```

Comment:
This is being removed, because it's not really needed and it expects `react-native` to be located in the `./node_modules` relative to the app.